pkg/config/config_template.yaml

```diff
@@ -3167,6 +3167,44 @@ api_key:
     #
     # stop_timeout: 5.0
 
+  ## @param netflow - custom object - optional
+  ## This section configures NDM NetFlow (and sFlow, IPFIX) collection.
+  #
+  # netflow:
+
+    ## @param enabled - boolean - optional - default: false
+    ## Set to true to enable collection of NetFlow traffic.
+    #
+    # enabled: true
```

Comment:
```suggestion
    # enabled: false
```
?

Reply:
@FlorianVeaux the value here is not necessarily the default but the suggested example

So, I think it's good to suggest `enabled: true` since it will allow the customer to just uncomment this line to enable netflow feature.

Reply:
This doesn't seem like the standard way of doing things in the configuration.
Searching for "enabled" in this example yaml file, _all_ options choose to display the default and not a recommendation in particular (well except one config which is deprecated).

Traps for instance take that approach, and I think it's worth keeping a standard between our options if possible

Reply:
Good, no strong opinion on this, let's make it match the default then.

Reply:
Fixed here https://github.com/DataDog/datadog-agent/pull/12508/commits/5ce4fcb8e52b81f60590620f382a4eb7c374ec55